docs/examples/basic_conceptual_guide/Part_4-Inference_Acceleration/Readme.md

```diff
@@ -0,0 +1,189 @@
+
+Acceleration recommendations depend on two main factors:
+* **Type of Hardware**: Triton users can choose to run models on GPU or CPU. Owing to the parallelism they provide, GPUs provide many avenues of performance acceleration. Models using PyTorch, TensorFlow, ONNX runtime, and TensorRT can utilize these benefits. For CPUs Triton 
+* **Type of the model**: Usually users leverage one or more of three different classes of model: `Shallow models` like Random Forests, `Neural Networks` like BERT or CNNs, and lastly, `Large Transformer Models` which are usually too big to fit in a single GPU's memory. Each of these category of models leverage different optimizations to accelerate models. 
+
+![Decision Tree](./img/2.PNG)
+
+With these broad categories in considered, let's drill down into the specific scenrios and decision making process to pick the most appropriate Triton Backend for the use case along with a brief discussion about possible optimizations. 
+
+## GPU Based Acceleration
+
+As mentioned before, acceleration for deep learning models can be achived with a wide variety of methods. Graph level optimizations like fusing layers can reduce the number of GPU kernels that are needed to be launched for execution. Fusing them together also makes this process more memory efficient and increases the density of operations. Once fused, we can utilize a kernel auto tuner to pick the correct combination of kernels to maximize utilization of GPU resources. Use of lower precision with techniques like quantization can drastically reduce memory requirements and increase througput. The exact nature of the tactics which can be used to optimize performance differ from GPU to GPU owing to different hardware design. These are a few of many challenges we solve for Deep Learning Practicioners with [NVIDIA TensorRT](https://developer.nvidia.com/tensorrt) which is an SDK focused on deep learning inference optimizations.
+
+While TensorRT works with popular deep learning with PyTorch, TensorFlow, MxNET, ONNX Runtime and more it also has framework level integrations with PyTorch([Torch-TensorRT](https://github.com/pytorch/TensorRT)) and TensorFlow([TensorFlow-TensorRT](https://github.com/tensorflow/tensorrt)) to provide their respective developers with the flexibility and fallback mechanisms. 
+
+### Using TensorRT directly
+
+There are three routes for users to use to convert their models to TensorRT: the C++ API, Python API, and trtexec (TensorRT's command line tool). [Refer this guide for a fleshed out example](https://github.com/NVIDIA/TensorRT/tree/main/quickstart/deploy_to_triton). That said there are two main steps needed. First, convert the model to a TensorRT Engine. It is recommended to use the [TensorRT Container](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/tensorrt) to run the command.
+
+```
+trtexec --onnx=model.onnx \
+        --saveEngine=model.plan \
+        --explicitBatch \
+        --useCudaGraph
```

Comment:
Explain why we're using `useCudaGraph` or remove it. Cuda Graphs don't seem to be mentioned anywhere in the doc.